File: ClientApp/src/app/models/orderProduct.ts
export class OrderProduct {
  Id: string;
  Quantity: number;
  Name: string;
  Price: number;
  Image: string;
  Measure: string;
 
  constructor(Id?: string, Quantity?: number, Name?: string,  Price?: number, Image?: string, Measure?: string) {
    this.Id = Id;
    this.Name = Name;
    this.Quantity = Quantity;
    this.Price = Price;
    this.Image = Image;
    this.Measure = Measure;



  }

}

File: Models/OrderProduct.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JUSTMOVE.Models
{
    public class OrderProduct
    {
        public string Id { get; set; }
        public string Quantity { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public string Image { get; set; }
        public string Measure { get; set; }
    }
}
